internal/targets/websites.go

```diff
@@ -73,6 +73,8 @@ var TargetWebsites = map[string]struct{}{
 	"https://foros.miranda-media.ru":         {},
 	"https://chernomorskoe.miranda-media.ru": {},
 	"https://kirovskoe.miranda-media.ru":     {},
+	"https://savelife.pw":                    {},
+	"http://ukraine-war.pw":                  {},
```

Comment:
What is `ukraine-war.pw`? I cannot find information about this link.

Reply:
haven't seen this before, but this is 99% scam and must be blocked. 
1. Domain was registered few days ago - https://www.whois.com/whois/ukraine-war.pw
2. Only cryptocurrency is acceptable and wallets are different than on original page
3. They state, that `Bank accounts are temporarily blocked` which is not true for REAL https://savelife.in.ua/donate/ page

Reply:
same is true for `savelife.pw`. I believe those must be shut down ASAP

Reply:
When you go to savelife.pw and select English language for the site you get sent to ukraine-war.pw. I asked about 
ukraine-war.pw on Telegram (Ukraine IT Army) but got no response so far. As for me I can't tell whether ukraine-war.pw is fake or not. IT has different IP than savelife.pw. It may be prudent to wait for more info on this target from the IT Army.

Reply:
Not sure about `http://savelife.pw/`. Also registered quite recently, but wallets seems be correct (at least for now)

Reply:
for sure we can add at least `http://ukraine-war.pw/` for now and wait a bit with `http://savelife.pw/`

Reply:
> haven't seen this before, but this is 99% scam and must be blocked.
> 
> 1. Domain was registered few days ago - https://www.whois.com/whois/ukraine-war.pw
> 2. Only cryptocurrency is acceptable and wallets are different than on original page
> 3. They state, that `Bank accounts are temporarily blocked` which is not true for REAL https://savelife.in.ua/donate/ page
So `savelife.in.ua` is the real one? I compared cryptocurrency addresses and they are different Bitcoin addresses. Ethereum addresses are the same, though - but that may be a trap. I find it also suspicious that they use all
so many altcoins like Tron. Strange thing that the Bitcoin address used does not start with bc1, which in turn means I guess that the address is pre-segwit (higher fees paid).